hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestObjectStoreV1.java

```diff
@@ -78,107 +90,143 @@ public static void init() throws Exception {
             .setOmId(omId)
             .build();
     cluster.waitForClusterToBeReady();
+    // create a volume and a bucket to be used by OzoneFileSystem
+    OzoneBucket bucket = TestDataUtil.createVolumeAndBucket(cluster);
+    volumeName = bucket.getVolumeName();
+    bucketName = bucket.getName();
+
+    String rootPath = String.format("%s://%s.%s/",
+            OzoneConsts.OZONE_URI_SCHEME, bucket.getName(),
+            bucket.getVolumeName());
+    // Set the fs.defaultFS and start the filesystem
+    conf.set(CommonConfigurationKeysPublic.FS_DEFAULT_NAME_KEY, rootPath);
+    // Set the number of keys to be processed during batch operate.
+    conf.setInt(OZONE_FS_ITERATE_BATCH_SIZE, 5);
+    fs = FileSystem.get(conf);
+  }
+
+  @After
+  public void tearDown() throws Exception {
+    deleteRootDir();
+  }
+
+  /**
+   * Cleanup files and directories.
+   *
+   * @throws IOException DB failure
+   */
+  private void deleteRootDir() throws IOException {
+    Path root = new Path("/");
+    FileStatus[] fileStatuses = fs.listStatus(root);
+
+    if (fileStatuses == null) {
+      return;
+    }
+
+    for (FileStatus fStatus : fileStatuses) {
+      fs.delete(fStatus.getPath(), true);
+    }
+
+    fileStatuses = fs.listStatus(root);
+    if (fileStatuses != null) {
+      Assert.assertEquals("Delete root failed!", 0, fileStatuses.length);
+    }
   }
 
   @Test
   public void testCreateKey() throws Exception {
-    String volumeName = "volume" + RandomStringUtils.randomNumeric(5);
-    String bucketName = "bucket" + RandomStringUtils.randomNumeric(5);
     String parent = "a/b/c/";
```

Comment:
@rakeshadr , can you explain the change included in this unit test? I'm not directly get this.

Reply:
Following are the changes done in TestObjectStore.
1) Removed the logic of directly iterating over the file and openfile tables. I've replaced iterator with `table.get(key)`, this will fetch from the table cache, if exists.  Like I described, with ratis enabled(edit logs), table iterator won't have most recent entries and only table cache contains the most recent entries. This was the reason for test case failure. As part of HDDS-4498, now by default ratis is enabled.
2) `ClientID` is needed for preparing openFileKey, the open file format is <parentID>/fileName/<clientID>. So, I have exposed `keyOutputStream.getClientID();` method by adding @VisibleForTesting.
3) I'm creating volume and bucket `TestDataUtil.createVolumeAndBucket(cluster)` at the beginning of the test instead of creating at every unit test case in this class. This is basically for the cleanup convenience and you can see `tearDown()` has been added to delete all the dirs from the root.

Reply:
Thanks @linyiqun for the reviews.

Reply:
Make the sense to me.